### PR TITLE
ISSUE-60: View list of Regex patterns added by user

### PR DESCRIPTION
**Pull request by alimuhammadahmer**
This is a pull request which contains Ruuter and Resql Endpoints to list regexes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,5 +11,19 @@ https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Resql/insert_regex.s
...
+``` 
+
+### View list of Regex patterns added by user
+
+https://github.com/buerokratt/Data-Anonymizer/issues/60
+
+https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Resql/list_regex.sql
+
+```
+```
```
Comment (turnerrainer) [inline README.md:24]: Please add a sample curl request.

File: README.md [hunk 2]
```diff
@@ -11,5 +11,19 @@ https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Resql/insert_regex.s
...
+
+https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Ruuter/GET/regex.yml
+
 ```
 ```
```
Comment (turnerrainer) [inline README.md:29]: Please add a sample curl request.

**Review by turnerrainer — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (alimuhammadahmer) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (alimuhammadahmer) [reply README.md]: Done

File: DSL.Resql/list_regex.sql [hunk 3]
```diff
@@ -0,0 +1 @@
+select Regex_Entity.id, Regex_Entity.entity, Regex_Entity.regex from Regex_Entity join (select Regex_Entity."id", max(Regex_Entity."created_at") as most_recent_record from Regex_Entity group by id) p ON (Regex_Entity.id = p.id and Regex_Entity.created_at = p.most_recent_record) where Regex_Entity.active = True;
```
Comment (turnerrainer) [inline DSL.Resql/list_regex.sql:1]: No newline at end of file

**Review by turnerrainer — APPROVED** [review]
No newline at end of file, otherwise OK

**Review by turnerrainer — CHANGES_REQUESTED** [review]
No newline at end of file, otherwise OK

File: DSL.Resql/list_regex.sql (hunk 3 above)
Comment (alimuhammadahmer) [reply DSL.Resql/list_regex.sql:1]: Done

File: README.md (hunk 2 above)
Comment (alimuhammadahmer) [reply README.md]: @turnerrainer please resolve this conversation

**Review by turnerrainer — APPROVED** [review]

**Pull request merged**